packages/openapi-generator/src/file-serializer/docs.ts

```diff
@@ -0,0 +1,71 @@
+import { EOL } from 'os';
+import { documentationBlock } from '@sap-cloud-sdk/util';
+import {
+  OpenApiApi,
+  OpenApiDocument,
+  OpenApiOperation,
+  OpenApiParameter,
+  OpenApiRequestBody
+} from '../openapi-types';
+import { serializeSchema } from './schema';
+
+export function apiDocumentation(
+  api: OpenApiApi,
+  document: OpenApiDocument
+): string {
+  return documentationBlock`
+  Representation of the ${api.name} API.
```

Comment:
`api.name` already ends with `Api`. So it would always be "Representation of the SomeApi API."
But now thinking of it, I don't have a better idea. If you don't have either feel free to just close this one.